docs/source/documentation.rst

```diff
@@ -328,42 +328,86 @@ Before computing the morphometrics of an image, make sure it has been segmented
 
     axondeepseg_morphometrics -i test_segmentation/test_sem_image/image1_sem/77.png -f axon_morphometrics 
 
-This generates a **'axon_morphometrics.xlsx'** file in the image directory::
+This generates a **'77_axon_morphometrics.xlsx'** file in the image directory::
```

Comment:
On line 323 above. I would say "If name of the excel file is not provided, the morphometrics will be saved as the original filename with suffix axon_morphometrics.xlsx".

Reply:
Added a note for the same in the latest commit fix.